toqito/matrix_ops/inner_product.py

```diff
@@ -36,7 +36,7 @@ def inner_product(v1: np.ndarray, v2: np.ndarray) -> float:
     :param v1: v1 and v2, both vectors of dimenstions :math:`(n,1)` where :math:`n>1`.
     :param v2: v1 and v2, both vectors of dimenstions :math:`(n,1)` where :math:`n>1`.
     :return: The computed inner product.
-    """
+    """  # noqa: E501
```

Comment:
Out of curiouisity I wonder why this is being flagged here. It's for a line that's too long, but there doesn't seem to be anything too long here?

Reply:
It's not flagged there. This is not ignored if it is added inside a docstring. For now, I asked `ruff` to ignore that docstring due to lines 12 and 20.